R/Figures.R

```diff
@@ -864,30 +864,62 @@ TADA_TwoCharacteristicScatterplot <- function(.data, id_cols = "TADA.ComparableD
   }
 
   if (!"TADA.ComparableDataIdentifier" %in% id_cols) {
-    print("Note: TADA.ComparableDataIdentifier not found in id_cols argument and is highly recommended.")
+    stop("Note: TADA.ComparableDataIdentifier not found in id_cols argument.")
   }
 
   # check that groups are in id_cols
   id <- unlist(unique(.data[, id_cols]))
   if (any(!groups %in% id)) {
     stop("The 'groups' vector contains one or more inputs that are not found within your input dataset. Check spelling and try again.")
   }
+  
+  # check that groups are length two for comparing two characteristics
```

Comment:
Reword comment? Do the groups need to be two characteristics? Couldn't you be comparing one characteristic at two locations?